src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,35 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import usersFromServer from '../../api/users';
+
+export type User = {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+};
+
+export type TodoItemType = {
+  id: number;
+  title: string;
+  userId: number;
+  completed: boolean;
+};
+
+type TodoInfoProps = {
+  todo: TodoItemType;
+};
+
+export const TodoInfo: React.FC<TodoInfoProps> = ({ todo }) => {
+  const user = usersFromServer.find((userItem) => userItem.id === todo.userId);
+
+  return (
+    <article data-id={todo.id} className={`TodoInfo ${todo.completed ? 'TodoInfo--completed' : ''}`}>
```

Comment:
better to use classNames package for such cases

Reply:
[DEMO LINK](https://aniashev.github.io/react_add-todo-form/)